src/app/hooks/use-accordion.ts

```diff
@@ -1,17 +1,37 @@
-import { useCallback, useState } from "react";
+import { useCallback, useEffect, useState } from "react";
 
-export const useAccordion = () => {
-	const [isExpanded, setIsExpanded] = useState(true);
+const getStorageKey = (key: string) => `accordion_${key}_isExpanded`;
+
+export const useAccordion = (key: string) => {
+	const storageKey = getStorageKey(key);
+	const [isExpanded, setIsExpanded] = useState<boolean>(() => {
+		const storedValue = localStorage.getItem(storageKey);
+		return storedValue ? JSON.parse(storedValue) : true;
```

Comment:
I think you can remove the `JSON.parse` part, because it stores only bool primitives which don't require deserialization, unless your plan was keeping data in a single key.

![image](https://github.com/user-attachments/assets/49602c0c-8813-4a52-84f7-6a0bce8dcbcb)

Reply:
In this case storing 1 or 0 could resolve the issue :eyes: